processing/src/main/java/io/druid/segment/FloatDimensionMergerV9.java

```diff
@@ -55,7 +55,12 @@ public FloatDimensionMergerV9(
   private void setupEncodedValueWriter(SegmentWriteOutMedium segmentWriteOutMedium) throws IOException
   {
     final CompressionStrategy metCompression = indexSpec.getMetricCompression();
-    this.serializer = FloatColumnSerializer.create(segmentWriteOutMedium, dimensionName, metCompression);
+    this.serializer = IndexMergerV9.createFloatColumnSerializer(
```

Comment:
Align with double version (there is a comment)

Reply:
done.